### PR TITLE
manifest: add --no-hostonly and VMWare storage drivers to initramfs

### PR DESCRIPTION
**Pull request by imcleod**

### CODE REVIEW

**Comment by jlebon** [comment]
Looks sane to me. Were you able to build with this fine?

File: fedora-coreos-base.yaml [hunk 1]
```diff
@@ -5,6 +5,12 @@ include: minimal.yaml
 boot_location: new
 tmp-is-dir: true
 
+# Add a richer set of early boot drivers, including vmware storage
+initramfs-args:
+  - "--no-hostonly"
```
Comment (dustymabe) [inline fedora-coreos-base.yaml]: weird - were we not doing `--no-hostonly` before in FCOS?

**Comment by dustymabe** [comment]
one question I asked in another place: should we try to get this change into dracut itself upstream rather than just carrying it here?

File: fedora-coreos-base.yaml (hunk 1 above)
Comment (imcleod) [reply fedora-coreos-base.yaml]: It's in Fedora Atomic Host I believe:

https://pagure.io/fedora-atomic/blob/master/f/fedora-atomic-host.json#_14

May have gotten lost in the shuffle.

**Comment by imcleod** [comment]
> one question I asked in another place: should we try to get this change into dracut itself upstream rather than just carrying it here?

Happy to pursue it there but I expect that will be a bit slower of a change to trickle out.  I appreciate "let's just patch it here" is not a great argument....

**Comment by dustymabe** [comment]
> Happy to pursue it there but I expect that will be a bit slower of a change to trickle out. I appreciate "let's just patch it here" is not a great argument....

How about a compromise? We can merge this here (quick fix), but can we open a PR there to see if we can make progress on the "long term" solution too?

File: fedora-coreos-base.yaml (hunk 1 above)
Comment (jlebon) [reply fedora-coreos-base.yaml]: It's the default in rpm-ostree, so we were still using it in FCOS: https://github.com/projectatomic/rpm-ostree/blob/b889a510560e0866fac94f7de06c258546c23ce7/src/libpriv/rpmostree-postprocess.c#L419

**Comment by imcleod** [comment]
> > Happy to pursue it there but I expect that will be a bit slower of a change to trickle out. I appreciate "let's just patch it here" is not a great argument....
> 
> How about a compromise? We can merge this here (quick fix), but can we open a PR there to see if we can make progress on the "long term" solution too?

SGTM - I'm perusing the dracut code at the moment

**Comment by cgwalters** [comment]
LGTM

**Comment by dustymabe** [comment]
> SGTM - I'm perusing the dracut code at the moment

+1 - running this locally now to make sure things don't blow up.

**Pull request merged**